mmv1/third_party/terraform/services/compute/go/resource_compute_forwarding_rule_test.go.tmpl

```diff
@@ -191,6 +191,7 @@ func TestAccComputeForwardingRule_forwardingRuleVpcPscExampleUpdate(t *testing.T
 				ResourceName:      "google_compute_forwarding_rule.default",
 				ImportState:       true,
 				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{"labels", "terraform_labels"},
```

Comment:
If labels are set in a post_create function why do we still need to ignore labels in the State verify? These should be set? I worry that this is a implication that they are not.

Reply:
The resource `google_compute_forwarding_rule` applies the new labels model. With the new labels model, the `labels` and `terraform_labels` fields are decided based on state. After importing, `labels` and `terraform_labels` will be empty as no value for these two fields in the state. 

https://github.com/hashicorp/terraform-provider-google/blob/main/google/services/compute/resource_compute_forwarding_rule.go#L1390

https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown?plain=1#L226

Reply:
Makes sense. Thank you for explaining.